src/components/attendee/NavOverlay/SearchOverlay/SearchOverlay.module.scss

```diff
@@ -0,0 +1,79 @@
+@use "scss/attendee/variables.scss"as *;
+@use "scss/attendee/font.scss";
+@use "scss/attendee/space";
+@use "scss/attendee/theme";
+
+.searchOverlayWrapper {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  max-width: 800px;
```

Comment:
[NB] Maybe we should export this as well. `$searchOverlayMaxWidth` or something similar.

Reply:
There is a file `./src/scss/attendee/layout.scss` but the intent is for a very low number of shared constants that general layout depends on. In all other cases a `$-private-constant` would be the best way to not pollute global nor imported module name space.

Reply:
[2ebc1f3](https://github.com/sparkletown/sparkle/pull/2827/commits/2ebc1f31eb8b1121e844036f240cd927eeb6d714)